dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java

```diff
@@ -498,4 +514,11 @@ protected Predicate getWorkflowPredicate(DescriptorLanguage descriptorLanguage,
         }
         return predicate;
     }
+
+    static Predicate addAuthorClauseToCriteriaBuilder(CriteriaBuilder cb, Root<?> entryRoot, Subquery<Author> subQuery) {
+        Root<Author> authorRoot = subQuery.from(Author.class);
+        final Predicate notEmpty = cb.isNotNull(
+                subQuery.select(authorRoot).where(cb.and(cb.equal(authorRoot.get("versionid"), entryRoot.get("actualDefaultVersion")))));
+        return cb.and(cb.isNotNull(entryRoot.get("actualDefaultVersion")), notEmpty);
```

Comment:
Why is the query for authors limited to the `actualDefaultVersion`? Is it because all the other filters are against workflow-level properties, but there is no workflow level property for authors?

Reply:
Yes, so authors could vary between workflow versions. I think to be consistent with old behaviour, we're just checking one of the versions (the default version)

Reply:
(assuming comment at https://github.com/dockstore/dockstore/pull/5706/files#diff-094b77f6c978c409e40b34fa5772f27ebf91e45fc3f0496745056ef1eb6f6d1eL167 is correct)

Reply:
> there is no workflow level property for authors?

Technically, we surface an entry-level authors property that is the default version's authors https://github.com/dockstore/dockstore/blob/83095ae11cd2b8f5f519c4b32689190f3855d09f/dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java#L900-L906